llvm/test/Transforms/InstCombine/ptr-replace-alloca.ll

```diff
@@ -429,9 +429,13 @@ entry:
 
 declare i8 @readonly_callee(ptr readonly nocapture)
 
+; FIXME: This should be able to fold to call i8 @readonly_callee(ptr nonnull @g1)
 define i8 @call_readonly_remove_alloca() {
```

Comment:
Change function name as well

Reply:
The name still expresses the intent of the test, not the actual result so I don't think it makes sense to change it